tests/unit/test_messages_integration.py

```diff
@@ -625,36 +741,52 @@ def test_logging_when_verboseish(capsys, logger, mode):
     assert_outputs(capsys, emit, expected_err=expected, expected_log=expected)
 
 
-@pytest.mark.parametrize(
-    "mode",
-    [
-        EmitterMode.QUIET,
-        EmitterMode.BRIEF,
-    ],
-)
-def test_initial_messages_when_quietish(capsys, mode, monkeypatch, tmp_path):
-    """Check the initial messages are sent when setting the mode to more quiet modes."""
+def test_initial_messages_when_quiet_mode(capsys, monkeypatch, tmp_path):
```

Comment:
pedantic: remove `when` from test function names

Reply:
Done, here and in a bunch of other places.